.gitlab-ci.yml

```diff
@@ -64,6 +63,13 @@ configure_python:
     paths:
       - ${PYTHON_ENVIRONMENT_PATH}
 
+.venv:
```

Comment:
What is this section for?

Reply:
This is the `extension` to add to any job that needs to use the python virtual environment.

Reply:
This used to be directly in the quartz job definition.
An aborted fix attempt had me duplicating that job, so I decided to share the virtual env.

This remains, because it seems to be a better design choice, although it may appear unrelated to the PR.

Reply:
Okay, thanks.  That makes sense.